packages/web-ui/src/ds/atoms/CodeBlock/SyntaxHightlighter/index.tsx

```diff
@@ -1,73 +1,29 @@
 'use client'
 
-import React, {
-  lazy,
-  ComponentProps,
-  Suspense,
-  useEffect,
-  useState,
-  useMemo,
-} from 'react'
-import { cn } from '../../../../lib/utils'
-import { CurrentTheme } from '../../../../constants'
+import { ComponentProps, Suspense, lazy } from 'react'
+import { Prism as PrismHighlighter } from 'react-syntax-highlighter'
+import { Skeleton } from '../../Skeleton'
 
-const LazyPrism = lazy(() =>
-  import('react-syntax-highlighter').then((mod) => ({ default: mod.Prism })),
-)
-
-type PrismThemeStyle = { [key: string]: React.CSSProperties } | undefined
-
-type Props = Omit<ComponentProps<typeof LazyPrism>, 'style'> & {
-  currentTheme: string | undefined
+type Props = Omit<ComponentProps<typeof PrismHighlighter>, 'style'> & {
+  currentTheme?: string
 }
 
+const LazyPrism = lazy(() => import('./LazyPrism'))
+
 export function SyntaxHighlighter({
   children,
   className,
   language,
   currentTheme,
 }: Props) {
-  const [style, setStyle] = useState<PrismThemeStyle>(undefined)
-  useEffect(() => {
-    let active = true
-
-    const loadTheme = async () => {
-      let themeModule
-      if (currentTheme === CurrentTheme.Dark) {
-        themeModule = await import(
-          'react-syntax-highlighter/dist/esm/styles/prism/one-dark'
-        )
-      } else {
-        themeModule = await import(
-          'react-syntax-highlighter/dist/esm/styles/prism/one-light'
-        )
-      }
-
-      if (active) {
-        setStyle(themeModule.default)
-      }
-    }
-    loadTheme()
-
-    return () => {
-      active = false
-    }
-  }, [currentTheme])
-
   return (
-    <Suspense fallback={<div className='p-4 text-sm'>Loading code...</div>}>
+    <Suspense fallback={<Skeleton height='h5' />}>
```

Comment:
I mean, this is using the skeleton. But it is not nice 😂 It does not represent the shape of a code block. 

Take, as example main editor loading state